test/testcafe/framework/page-objects/PollingPageObject.js

```diff
@@ -12,6 +13,14 @@ export default class PollingPageObject extends BasePageObject {
     return this.form.getTitle();
   }
 
+  getContent() {
```

Comment:
already exists in BaseFormObject or move into that?

Reply:
The selector in basePageObject is for getIonMessages but here the text in poll view is added manually.